docs/_v1_0/8_langtidslagring.md

```diff
@@ -8,6 +8,6 @@ Posten signering tilbyr en tilleggstjeneste for langtidslagring av originaldokum
 
 For avsendere med langtidslagring aktivert, lagres alle dokumenter i minst __50 år__. Dersom langtidslagring _ikke_ er aktivert kan dokumentene bli utilgjengeliggjort etter __30 dager__.
 
-Avsendere som bruker [avsenderportalen](#org-portal) kan aksessere langtidslagrede dokumenter via webgrensesnittet. Dokumentene er tilgjengelig via REST-grensesnittet for avsendere som integrerer mot tjenesten maskinelt, som beskrevet i [teknisk dokumentasjon](https://github.com/digipost/signature-api-specification/integrasjon/README.md#tilleggstjeneste-for-langtidslagring).
+Avsendere som bruker [avsenderportalen](#org-portal) kan aksessere langtidslagrede dokumenter via webgrensesnittet. Dokumentene er tilgjengelig via REST-grensesnittet for avsendere som integrerer mot tjenesten maskinelt, som beskrevet i [teknisk dokumentasjon](https://github.com/digipost/signature-api-specification/tree/master/integrasjon#tilleggstjeneste-for-langtidslagring).
```

Comment:
Eller bør den peke på https://github.com/digipost/signature-api-specification/blob/master/integrasjon/README.md#tilleggstjeneste-for-langtidslagring? Hva synes du @fredva – det spiller kanskje ingen rolle? 🤷‍♂️

Reply:
Whoops, her glapp det fra min side opprinnelig ja. Fiksen er fin sånn den er nå synes jeg.

Reply:
🙆‍♂️